extension/services/soundcloud/player.js

```diff
@@ -0,0 +1,49 @@
+/* globals helpers */
+
+this.player = (function() {
+  // Play button Reference
+  const SEARCH_PLAY = ".sound__header a[title='Play']";
+
+  class Player extends helpers.Runner {
+    action_play() {
+      const button = this.querySelector("button[aria-label='Play current']");
```

Comment:
Ideally these would use selectors that didn't use these English phrases, as they will break in other languages. There's several open bugs in Spotify for the same issues. (Of course I can see why you'd select these, because I did offer up Spotify as a good example...)

If it's not easy to figure out other selectors we can do a followup bug to fix these.

Reply:
removed selectors which used English phrases and added selectors based on class names.